common/src/main/java/gov/cms/ab2d/common/dto/ContractDTO.java

```diff
@@ -17,4 +21,17 @@
 
     @EqualsAndHashCode.Exclude  // contractNumber is sufficient, breaks on Windows due sub-seconds not matching
     private String attestedOn;
+
+    public void setAttestedOn(String offsetDateTime) {
+        attestedOn = offsetDateTime;
+    }
+    public void setAttestedOn(OffsetDateTime offsetDateTime) {
+        attestedOn = offsetDateTime.toString();
```

Comment:
Instead, let's make attestedOn the OffsetDate.  This is code debt from previous work that is making this PR more complicated than needed.